<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui::ModuleView</class>
 <widget class="QWidget" name="gui::ModuleView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>491</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QScrollArea" name="parameterScroller">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>2</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>200</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="gui::Parameters" name="parameters">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>430</width>
           <height>252</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QTextBrowser" name="messages">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>50</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonLayout">
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::StandardButton::NoButton</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="messagesLabel">
         <property name="text">
          <string>Output:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="messagesButtonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::StandardButton::NoButton</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>gui::Parameters</class>
   <extends>QWidget</extends>
   <header>parameters.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
